lib/components/map/base-map.js

```diff
@@ -248,8 +248,8 @@ class BaseMap extends Component {
 
   render () {
     const { config, children, diagramLeg, elevationPoint, popupLocation } = this.props
-    const { baseLayers } = this.props.config.map
-
+    const { baseLayers } = config.map
+    const showElevationProfile = Boolean(config.elevationProfile)
```

Comment:
Can you make a PR to https://github.com/conveyal/trimet-mod-otp/ with this added config setting enabled?

Reply:
https://github.com/conveyal/trimet-mod-otp/pull/186